pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[build-system]
+requires = [
+    "setuptools>=60",
+    "setuptools-scm>=8.0",
+]
+
+[tools.setuptools.dynamic]
+version = {attr = "ome_zarr.__version__"}
+
+[project]
+name = "ome-zarr"
+readme = "README.rst"
+dynamic = ["version"]
+description="Implementation of images in Zarr files."
+
+requires-python = ">3.8"
+
+dependencies = [
+    "dataclasses;python_version<'3.7'",
+    "tifffile<2020.09.22;python_version<'3.7'",
```

Comment:
Do we need the `<'3.7'` support here, since we have `requires-python = ">3.8"`

Reply:
No! Thanks. Let's get rid of these.